docs/development/react/how-to-deploy-a-react-app-on-ubuntu-18-04/index.md

```diff
@@ -0,0 +1,193 @@
+---
+author:
+  name: Linode
+description: This guide will show you how to deploy a React app to a Ubuntu 18.04 Linode that is running a web server. You will configure your Linode to host a React app by installing and configuring a web server, like Apache or NGINX. Then, you will configure your computer to ensure you can develop a React app locally. Finally, you will deploy all of your site's build files to your remote Linode using Rsync.
+og_description: This guide will show you how to deploy a React app to a Ubuntu 18.04 Linode that is running a web server. You will configure your Linode to host a React app by installing and configuring a web server, like Apache or NGINX. Then, you will configure your computer to ensure you can develop a React app locally. Finally, you will deploy all of your site's build files to your remote Linode using Rsync.
+keywords: ['react','reactjs','deploy','rsync']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2018-04-24
+modified: 2020-04-24
+modified_by:
+  name: Linode
+title: "How to Deploy a React Application on Ubuntu 18.04"
+h1_title: "Deploying a React Application on Ubuntu 18.04"
+contributor:
+  name: Linode
+external_resources:
+- '[React - A JavaScript library for building user interfaces](https://reactjs.org/)'
+- '[Deploy a React App with Sass Using NGINX](http://zabana.me/notes/build-deploy-react-app-with-nginx.html)'
+audiences: ["beginner"]
+concentrations: ["Web Applications"]
+languages: ["javascript"]
+---
+## What is React?
+
+[React](https://reactjs.org/) is a popular JavaScript library for building user interfaces. While React is often used as a frontend for more complex applications, it's also powerful enough to be used for full client-side applications on its own.
+
+Since a basic React app is static (it consists of compiled HTML, CSS, and JavaScript files), it is easy to deploy from a local computer to a Linode using [Rsync](/docs/tools-reference/tools/introduction-to-rsync/). This guide shows how to set up your Linode and local machine so that you can easily deploy your app whenever changes are made. See the [More Information](#more-information) section of this guide for
```

Comment:
Complete the thought in the last sentence, and please bake the distro name into this paragraph somewhere.

Reply:
Last line should've been deleted. I've done that and added the distro to the opening paragraph.